code/modules/client/preference/preferences.dm

```diff
@@ -2000,8 +2000,10 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 				if("winflash")
 					windowflashing = !windowflashing
 
-				if("afk_fast_cryo")
-					afk_fast_cryo = alert(user, "Would you like to be qualified for the fast cryoing? Being AFK for 5 minutes will result in cryoing then (instead of [config.auto_cryo_afk])", "Fast cryoing", "Yes", "No") == "Yes"
+				if("afk_watch")
+					to_chat(user, "<span class='info'>You will now get put into cryo dorms after [config.auto_cryo_afk] minutes. \
+								Then after [config.auto_despawn_afk] minutes you will be fully despawned. You will receive a visual and auditory warning before you will be put into cryodorms.</span>")
+					afk_watch = !afk_watch
```

Comment:
Correct me if I am wrong, but this is going to show up whether you turn this thing on or off, making is a huge confusion

Reply:
Aye. Saw it when I pushed the changes. Updated now